app.js

```diff
@@ -13,6 +25,50 @@ app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
 
+app.get('/', (req, res) => {
+    res.render('home',)
+})
+
 // Our routes go here:
 
-app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
+app.get("/artist-search", (req, res) => {
+  const artist = req.params.artist
```

Comment:
Here you need to pass the query string. Right now it's not searching for any artist - the search form is not linked to anything. You need to use the query string and correctly name the form.